docs/services/thorlabs_cube_motor_kinesis.rst

```diff
@@ -7,6 +7,7 @@ This service uses bits of the official vendor Python library:
 https://github.com/Thorlabs/Motion_Control_Examples/tree/main/Python

```

Comment:
Could you reformat the two links in this paragraph so that both are marked properly when rendered? There are a couple of examples of links in the Allied Vision camera service doc, you can see how to format a url there. This is especially important for the other link, on line 9, since it contains a bunch of special characters.

Reply:
Also, I have no clue how to do this in an rst file, but maybe you can figure out how to put a single line break after each link? I don't mean a new paragraph, but just pushing the text to the next line.
This is one of those requests that if you cannot figure out in ~15-20 minutes, please just drop it...

Reply:
I managed to do it !